src/leds.c

```diff
@@ -0,0 +1,73 @@
+#include "leds.h"
+
+#include "ch.h"
+#include "hal.h"
+#include "board.h"
+
+enum LedState actLed;
+enum LedState statLed;
+enum LedState errLed;
```

Comment:
LedState is no new type, dont do typedefs (see linux codestyle). Better would be to place the leds states in a struct prefix with `g_*` and make it static.